### DISCUSSION TITLE
All is almost working...some follow-up issues I'm having

### DISCUSSION THREAD

**Discussion by maussimo010** [nzbgetcom/nzbget#309] [General]
I use No-IP to maintain a DDNS so I can use NZBget remotely rom my phone to control that as well as Sonarr/Radarr.

I cannot load NZBget now on the phone using 192.XXX or by using my DDNS address, because it asks me to provide a login. All password/password options as far as I can tell have been shut off in NZBget. I can load Radarr/Sonarr with no issues.

I left the ControlIP setting as 0.0.0.0  This all worked before flawlessly before I updated everything.

I looked at my old settings from v21, and under security/ControlIP I had my local address of 192.168.2.57 entered, no user/pass, and for authorizedIP I had 127.0.0.1 entered. I was able to reach the old version on my phone with no prompt for user/pass.

![IMG_5992](https://github.com/user-attachments/assets/79eab86d-2481-4fdd-bb11-2e40b62426c1)

![IMG_5990](https://github.com/user-attachments/assets/99c610c9-2b0c-4764-bfb6-2e47142b94a4)

![IMG_5989](https://github.com/user-attachments/assets/d270de0d-d184-4a2b-8593-9b11e9fd1f2a)

I then entered these same settings into v24, and It's back to asking for user/pass. Why is it asking if user/pass is blank on the security page?

If it says set empty value, does that mean to leave the field empty, or is there something that has to be entered into the field to tell the app that it is to be ignored or "empty"?

Hoping someone can help me with this part: In Radarr, I am getting this error:

You are using docker; download client NZBGet places downloads in /downloads/complete/Movies but this directory does not appear to exist inside the container. Review your remote path mappings and container volume settings.

I understand (I think) that I need to duplicate the container, go to the .yml page and use an existing project, but I am not a coder nor a hardcore computer guy, so I do not know how to add a line to where I can add this folder to the build. 

thoughts?

**Comment by luckedea** [comment]
@maussimo010 Looks like you had NZBGET_USER / NZBGET_PASS set in your docker-compose, so SECURITY tab settings aren't working.

So, please remove NZBGET_USER / NZBGET_PASS and rebuild.

**Reply by maussimo010** [reply]
Ok, that worked perfectly! 

Much appreciated!

my last issue at the moment is this:

![IMG_5994](https://github.com/user-attachments/assets/b27270e6-d7e7-48ac-a1a0-017bb101915d)

I think I understand what it’s saying, which is that I need to duplicate the build again, put that folder into file station, then map it in the build to that folder…BUT…I’m not sure how to write it out so that it maps to it. Idk what a colon : means after a location, I don’t know when to use them, or if they’re needed. I don’t even know what doing this is called aside from mapping to be able to google it to learn how to do it really.

it should look like this?

![IMG_5995](https://github.com/user-attachments/assets/ca00746d-b63b-4f5f-a340-6866c57bf507)

Also, I have a folder called video, and in that folder are two folders called movies and tv...those are folders I use for my actual movie and tv how library...are those the two folder is should be pointing/mapping to instead of creating movie/tv folders in nzbget? is nzbget by default trying to start a movie/tv library in the tv/movie folders that are supposed to be in the complete folder?
